xdc-plugin/tests/minilitex_ddr_arty_golden.json

```diff
@@ -0,0 +1,247 @@
+{
+  "IOBUF": {
+    "IN_TERM": "UNTUNED_SPLIT_40",
+    "IOSTANDARD": "SSTL135",
+    "LOC": "ddram_dq[0]:K5",
```

Comment:
This is not the LOC syntax from Vivado.  Can we use a new parameter, e.g. XLOC or LOC_PAIRS, or IO_LOC, etc?

Reply:
Done